.github/workflows/builds.yml

```diff
@@ -61,3 +61,31 @@ jobs:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
       - name: Update the latest tag
         uses: EndBug/latest-tag@latest
+  bully:
+    name: Bully
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the repository
+        uses: actions/checkout@v2
+      - name: Install the toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly-2021-03-25
+          override: true
+      - run: make evm-bully=yes
+      - run: ls -lH release.wasm
+      - name: Upload the release.wasm artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: bully.wasm
+          path: bully.wasm
+      - name: Update the latest release
+        uses: svenstaro/upload-release-action@v2
+        with:
+          tag: latest
+          file: bully.wasm
+          overwrite: true
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update the latest tag
```

Comment:
You need to remove the `Update the latest release` and `Update the latest tag` steps, lest you introduce a race condition for updating the `latest` tag.

In addition, it is also unlikely that you actually need the `Upload the release.wasm artifact` step (which also wasn't properly renamed).